Reviewer/src/main/java/com/softserveinc/reviewer/service/StatusService.java

```diff
@@ -0,0 +1,56 @@
+package com.softserveinc.reviewer.service;
+
+import java.net.ConnectException;
+import java.net.HttpURLConnection;
+import java.net.SocketTimeoutException;
+import java.net.URL;
+import java.util.concurrent.Callable;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.concurrent.Future;
+
+import com.softserveinc.reviewer.model.DataProvidersStatus;
+
+public class StatusService {
+    private final ExecutorService executorService = Executors.newFixedThreadPool(3);
+
+    public DataProvidersStatus getStatus() {
+        Future<String> switchBoardFuture = executorService.submit(new CheckStatus("http://localhost:8085/healthcheck"));
+        Future<String> oracleFuture = executorService.submit(new CheckStatus("http://localhost:8085/healthcheck"));
+        Future<String> elasticSearchFuture = executorService.submit(new CheckStatus("http://localhost:8085/healthcheck"));
+        try {
+            return new DataProvidersStatus(switchBoardFuture.get(), oracleFuture.get(), elasticSearchFuture.get());
```

Comment:
I'd love to have composed task from those three. Hence it would be possible to fo get result of this task with some guaranteed timeout https://docs.oracle.com/javase/7/docs/api/java/util/concurrent/Future.html#get(long,%20java.util.concurrent.TimeUnit)